packages/modules/fulfillment/integration-tests/__tests__/fulfillment-module-service/shipping-option.spec.ts

```diff
@@ -1059,7 +1072,7 @@ moduleIntegrationTestRunner<IFulfillmentModuleService>({
             )
 
             const types = await service.listShippingOptionTypes()
-            expect(types).toHaveLength(2)
+            expect(types).toHaveLength(4)
```

Comment:
question: Why did this change? Don't see any changes to seed data or similar 😄

Reply:
Dml was applying orphan removal on one to one which is pretty aggressive, since dml is completely locked up and do not provide such configuration, it seems dangerous to physically remove resources that are not removed by the user but as a side effect under the hood. So if you attach a new resource, the previous one is detached and the new one created and attached. We have similar behavior in other places as well. But with the dml we enforce the convention everywhere and align everything 👌

Reply:
Sounds good – is this something we should mention in the release notes?

Reply:
Yeah think this looks good. In the worst case, developers will end up with unassociated resources, which I think is better than mistakenly deleted them.